ops/jenkins/bfd-mgmt-synthea-generation-build/build_docker.sh

```diff
@@ -6,6 +6,9 @@ set -eo pipefail
 BUILD_CONTEXT_ROOT_DIR=$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")
 readonly BUILD_CONTEXT_ROOT_DIR
 
+SYNTHEA_SCRIPTS_ROOT_DIR="."
+readonly SYNTHEA_SCRIPTS_ROOT_DIR
```

Comment:
This is a little _marked_. What's going on here?

Reply:
So...Realizing that I marked this as RTR a bit prematurely, but yeah...Wanted to stay consistent with using `CONST`s for directories, but this is more of a code smell than anything else, let me clean this up